test/rmgpy/chemkinTest.py

```diff
@@ -926,21 +926,21 @@ def setUpClass(cls):
             """
                              Reaction index: Chemkin #2; RMG #4
                              Template reaction: H_Abstraction
-                             Estimated using template (C/H3/Cs;C_methyl) for rate rule (C/H3/Cs\H3;C_methyl)
+                             Estimated using template (C/H3/Cs;C_methyl) for rate rule (C/H3/Cs\\H3;C_methyl)
```

Comment:
Just for my curiosity, why does the \ have to be escaped now?

Reply:
Pending deprecation in future version of Python - `\H` is being interpreted as an escape character (but `\H` is _not_), being found invalid, and then treated as literal `\H`. In the future this will error, just fixing it now